test/irb/test_completion.rb

```diff
@@ -107,11 +107,56 @@ def test_complete_require_relative
     end
 
     def test_complete_variable
+      # Bug fix issues https://github.com/ruby/irb/issues/368
+      private_methods = ''
+      methods = ''
+      global_variables = ''
+      local_variables = ''
+      instance_variables = ''
       str_example = ''
-      str_example.clear # suppress "assigned but unused variable" warning
+      @str_example = ''
+
+       # suppress "assigned but unused variable" warning
```

Comment:
```suggestion
      # suppress "assigned but unused variable" warning
```

Can we fix this comment's indentation?